src/scripts/main.js

```diff
@@ -1 +1,78 @@
-'use strict';
+function showNotification(message) {
```

Comment:
The `showNotification` function is defined to accept only one parameter, `message`, but it is called with two arguments in the promise handlers. Consider updating the function definition to accept two parameters or adjust the calls to match the function signature.